internal/app/trustd/main.go

```diff
@@ -56,20 +58,34 @@ func main() {
 		log.Fatal(err)
 	}
 
+	domainName, err := ioutil.ReadFile("/proc/sys/kernel/domainname")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	dnsNames := []string{}
```

Comment:
Should _also_ include the short hostname in the set.

Reply:
👍 done